support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -168,4 +168,36 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		excludeCountriesSubjectToContributionsOnlyAmounts: false,
```

Comment:
Could I piggyback on this PR and add this change in this PR?
https://github.com/guardian/support-frontend/pull/6369#discussion_r1785992806

Reply:
@andrewHEguardian, yes I did think about just doing it here! I'll commit the change now